### DISCUSSION TITLE
Não é tão fácil quanto parece!!!!

### DISCUSSION THREAD

**Discussion by Jefferson-AF** [digitalinnovationone/dio-lab-open-source#45553] [Desafios DIO]
Tenho 42 anos, vim para cá pensando em aprender e quem sabe no futuro uma mudança para área de programação. O bootcamp cita Java, mas até agora não tem nada de Java, se eu quiser aprender alguma coisa preciso aprender Git, GitHub e contribuir para um projeto, mas nada de codificação até agora, não dá para pular essa parte e aprender, preciso fazer outras coisas antes.... o curso cita que NÃO é necessario conhecimento em programação, mas a didática dos instrutores não demonstra isso. Usam termos e jargões que quem chegou agora sem conhecimento anterior não tem. Não foi ensinado até agora nada. NADA de Java. Custava dizer que é necessario um conhecimento previo? Ou custava começar com noções de Java, antes de ensinar como compartilhar coisas no Github? COMEÇEM DO COMEÇO. A plataforma não esta fazendo isso. Me sinto enganado e desiludido. Se quiser aprender Java aqui não é o lugar, pelo menos isso é o que sinto. Eu como investidor não investiria nisso. Porque não dizer que é pra quem ja sabe ou então rever a proposta e começar com uma metodologia diferente. Porque essa não é para iniciantes.

[a comment by an automated account is not shown]

**Comment by AdaoElias** [comment]
Boa tarde, eu tenho 49 anos nunca gostei de programação, estou me aventurando em aprender. Já desisti por 2 vezes. E agora sinto a necessidade de mudar de carreira e esse Boo Camp creio que possa me proporcionar isso.
Tenho várias dificuldades, creio que repetindo e refazendo os processos é a forma mais coesa de aprender e fixar os conteúdos. 
Lembrando que o Github é a principal ferramenta para versionamento e controle de fluxo de trabalho.
Insista um pouco masi que vai valer a pena. depois vovê ver o tanto que já aprendeu.
Quanto as duvidas compartilhe com a comunidade que sempre tem alguém disposto a judar. 
No que for preciso estamos à disposição.
